consensus/istanbul/validator/weighted.go

```diff
@@ -665,8 +665,11 @@ func (valSet *weightedCouncil) Refresh(hash common.Hash, blockNum uint64, config
 		return nil
 	}
 
-	totalStaking, _ := calcTotalAmount(weightedValidators, newStakingInfo, stakingAmounts)
-	calcWeight(weightedValidators, stakingAmounts, totalStaking)
+	// weight and gini were neutralized after Kore hard fork
+	if !chainRules.IsKore {
+		totalStaking, _ := calcTotalAmount(weightedValidators, newStakingInfo, stakingAmounts)
+		calcWeight(weightedValidators, stakingAmounts, totalStaking)
+	}
```

Comment:
@mckim19 Don't we need to forcefully set the weight to 1 for all validators?

Under a hypothetical scenario,
- A chain started at Magma. 4 nodes have weights [4,3,2,1]. Saved in IstanbulSnapshot, etc.
- The chain passes the Kore block.
- The `weight` field never changes and still [4,3,2,1]? not [1,1,1,1]?

Reply:
Yes, so I add new function "setZeroWeight".
Thank you~!